test/integration/features/proxy.feature

```diff
@@ -0,0 +1,31 @@
+@proxy @linux
+Feature: Test the proxy
+
+  The user try to use crc behind proxy. he/she expect the
+  crc start happen successfully and able to deploy the app and check its
+  accessibility.
+
+  Scenario: Setup the proxy container using podman
+    Given executing "sudo podman run --name squid -d -p 3128:3128 quay.io/crcont/squid" succeeds
```

Comment:
maybe there are pre-made squid images which we could use?

Reply:
@cfergeau you mean instead maintaining `quay.io/crcont/squid` ourself?  I really want to control the image build ourself for any crc specific usage. I don't trust existing images, not sure what base image they chose and how often it updated. Another good thing maintaining ourself, if we need any change we just do it and rebuild then waiting for existing image maintainer.

Reply:
> I don't trust existing images, not sure what base image they chose and how often it updated. 

And we know we'll almost never update it, not sure it's better ;)

> Another good thing maintaining ourself, if we need any change we just do it and rebuild then waiting for existing image maintainer.

We always have the option to switch to our own image when we have such needs. And in the meantime, that's one less thing we have to take care of.